go/README.md

```diff
@@ -1,109 +1,12 @@
-This directory contains the Go logic that's executed by the `EmbeddedOnlineFeatureServer` from Python.
```

Comment:
Any reason we are removing the document here?

Reply:
That piece of code is not needed if we remove GOPY bindings.